examples/pytorch/stgcn_wave/README.md

```diff
@@ -0,0 +1,35 @@
+Spatio-Temporal Graph Convolutional Networks
+============
+
+- Paper link: [arXiv](https://arxiv.org/pdf/1709.04875v4.pdf)
+- Author's code repo: [https://github.com/VeritasYin/STGCN_IJCAI-18](https://github.com/VeritasYin/STGCN_IJCAI-18).
+Dependencies
+------------
+- PyTorch 1.1.0+
+- sklearn
+- dgl
+
+
+
+How to run
+----------
+please get METR_LA dataset from [https://drive.google.com/open?id=10FOTa6HXPqX8Pf5WRoRwcFnW9BrNZEIX](https://drive.google.com/open?id=10FOTa6HXPqX8Pf5WRoRwcFnW9BrNZEIX).
+and [https://github.com/chnsh/DCRNN_PyTorch](https://github.com/chnsh/DCRNN_PyTorch)

```

Comment:
BTW you don't have to write `[hyperlink](hyperlink)` for hyperlinks in GitHub READMEs.